src/hooks/useSearch.ts

```diff
@@ -20,12 +20,14 @@ async function getSearch(query = initialSearchCriteria) {
     },
   };
 
+  // TODO: remove this later when BE is updated
+  const url = `/searches${query.use_vespa ? "?use_vespa=true" : ""}`;
```

Comment:
Does this need to be `True` with an uppercase T?

Reply:
No, lower case was working fine when I tested this